modules/swagger-jaxrs2/src/main/java/io/swagger/v3/jaxrs2/Reader.java

```diff
@@ -449,6 +448,14 @@ public OpenAPI read(Class<?> cls, String parentPath) {
                     }
 
                     openAPI.setPaths(this.paths);
+
+                    io.swagger.v3.oas.annotations.Operation apiOperation =
+                            ReflectionUtils.getAnnotation(method, io.swagger.v3.oas.annotations.Operation.class);
+                    // requestBody
+                    if (apiOperation != null && apiOperation.requestBody() != null && operation.getRequestBody() == null) {
+                        OperationParser.getRequestBody(apiOperation.requestBody(), classConsumes, methodConsumes, components).ifPresent(
+                                requestBodyObject -> operation.setRequestBody(requestBodyObject));
+                    }
```

Comment:
This doesn't seem to handle `@RequestBody` as method level annotation

Reply:
Sure. Now I have added this functionality.